InformationScripting/src/query_framework/QueryResultVisualizer.cpp

```diff
@@ -113,6 +118,21 @@ Optional<int> QueryResultVisualizer::visualize(const TupleSet& ts)
 			item->addOverlay(overlay, HIGHLIGHT_OVERLAY_GROUP);
 		}
 	}
+
+	auto htmlTuples = ts.tuples("html");
+	if (htmlTuples.size()) {
+		QString htmlContent = (*htmlTuples.begin())["html"];
+		// TODO an overlay or something would probably be better for this:
+		// TODO position it a bit better? right now it is just centered.
+		auto browser = new Visualization::WebBrowserItem{nullptr, htmlContent};
```

Comment:
Could you please use the `frame` style here `WebBrowserItem::itemStyles().get("frame")`. Feel free to adjust the style to make it more visually appealing to you. :)